swagger2markup-core/src/main/resources/io/github/swagger2markup/lang/labels_zh.properties

```diff
@@ -22,8 +22,8 @@ name_column=\u540d\u79f0
 description_column=\u8bf4\u660e
 headers_column=\u5934
 scopes_column=\u4f5c\u7528\u57df
-produces=\u751f\u6210
-consumes=\u6d88\u8017
+produces=\u5e94\u7b54Content-Type
```

Comment:
Neither the source any other language has the words `Content-Type`, why is there a need for it?